src/app/features/asset-list/asset-list.tsx

```diff
@@ -78,15 +50,11 @@ export function BalancesList(): React.JSX.Element {
           />
         ) : null,
       })}
-      <CryptoCurrencyAssetItem
-        assetBalance={stxEffectiveBalance}
-        usdBalance={stxEffectiveUsdBalance}
-        address={address}
-        additionalBalanceInfo={stxAdditionalBalanceInfo}
-        additionalUsdBalanceInfo={stxAdditionalUsdBalanceInfo}
-        icon={<StxAvatar />}
-      />
-      <StacksFungibleTokenAssetList assetBalances={stacksFtAssetBalances} />
+
```

Comment:
Does `{account => <StacksBalanceItem account={account} />}` mean to only pass and render `<StacksBalanceItem` is `account` ?

Reply:
The only purpose of this is that the component returns `null` if there's no account, so we can only render `<StacksBalanceItem account={account} />}` when it's defined